.changeset/good-worms-heal.md

```diff
@@ -0,0 +1,13 @@
+---
+'@astrojs/markdoc': patch
+'astro': patch
+---
+
+The function
+`emitESMImage` has been deprecated, and it will be removed in a next major release of Astro. Use the function
+`emitImageMetadata` instead.
+
+```diff
+- import {emitESMImage} from "astro/assets/utils";
++ import {emitImageMetadata} from "astro/assets/utils";
+```
```

Comment:
``````suggestion
```

The new function was created to operate the same way as the old function, and should work interchangeably in your code. However, its signature is not identical and you may need to check that your code is working as intended.
``````
Given that this isn't an *identical* function, it might be good to include something like this. If you can be more specific about *what* people might need to check or *where* they might notice something breaking, that's even better!